src/types/error_responses.ts

```diff
@@ -0,0 +1,11 @@
+import { Registration } from './registration.types';
+
+export interface Problem {
```

Comment:
brukes ikke utenfor denne fila
```suggestion
interface Problem {
```